adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -918,6 +920,83 @@ def __call__(self, client, context, next_step):  # noqa: C901
         next_step(client, context)
 
 
+class CreateOrUpdateAssets(Step):
+    """Create or update assets in MPT based on Adobe Subscriptions."""
+
+    def __call__(self, client, context, next_step):
+        """Create or update assets in MPT based on Adobe Subscriptions."""
+        if not context.adobe_new_order_id:
+            next_step(client, context)
+            return
+
+        adobe_client = get_adobe_client()
+        one_time_skus = get_one_time_skus(client, context.order)
+        for line in filter(
+            lambda x: get_partial_sku(x["offerId"]) in one_time_skus,
+            context.adobe_new_order["lineItems"],
+        ):
+            order_line = get_order_line_by_sku(context.order, line["offerId"])
+            adobe_subscription = adobe_client.get_subscription(
+                context.authorization_id, context.adobe_customer_id, line["subscriptionId"]
+            )
+
+            if adobe_subscription["status"] != AdobeStatus.PROCESSED:
```

Comment:
It is a good idea to always use `.value` if Enum is used

Reply:
You only need `.value` in fixtures when you assign value